src/tensorlake/function_executor/handlers/run_function/handler.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         self._invocation_state: InvocationState = invocation_state
         self._logger = logger.bind(
             module=__name__,
-            graph_invocation_id=request.graph_invocation_id,
+            invocation_id=request.graph_invocation_id,
             task_id=request.task_id,
```

Comment:
Let's add a new optional field allocation_id to the request and add it to the logger here.